bip-musig2/sig_agg_vectors.json

```diff
@@ -0,0 +1,88 @@
+{
+	"keys": [
+		"487D1B83B41B4CBBD07A111F1BBC7BDC8864CFEF5DBF96E46E51C68399B0BEF6",
+		"4795C22501BF534BC478FF619407A7EC9E8D8883646D69BD43A0728944EA802F",
+		"0F5BE837F3AB7E7FEFF1FAA44D673C2017206AE836D2C7893CDE4ACB7D55EDEB",
+		"0FD453223E444FCA91FB5310990AE8A0C5DAA14D2A4C8944E1C0BC80C30DF682"
+	],
+	"pnonces": [
+		"0279BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F817980279BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798",
+		"0327EE3C4078B3EE8888C86980C9349B033360C041108412B076D64A199D7322470211A8415F0F7044FF94296CD3006D1E0BE7209F1549CF8F96D03C60ACA36DE69C",
+		"032D459FC804D89AE5B0352A92B4EAADB5AB2F8F5479C072D9BE4C9887A23CA9E40238A68BF96C2904BBF409527F3D8E1EE2D6A1F6A0F3E68016B3BB575EF22BA2DD",
+		"03202ED19F8FF57B795DA93D3DF40DC25EB4BBF5B0CCCD31E71CA2409A8C5575D602B73DFC004F0890B1F3BBF99A42746F02F1B6CB18D16AAD2F8703E779B4688ECA",
+		"020663C56E861775F206E0268F5DD517143B6F55374A5C499DB44AD30E342AB81C03F33D3F53D7528B3ADA8523C25C86FE4479FC3540CCED28D1D1BE351CF139AE64",
+		"0379BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F817980379BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798"
+	],
+	"tweaks": [
+		"B511DA492182A91B0FFB9A98020D55F260AE86D7ECBD0399C7383D59A5F2AF7C",
+		"A815FE049EE3C5AAB66310477FBC8BCCCAC2F3395F59F921C364ACD78A2F48DC",
+		"75448A87274B056468B977BE06EB1E9F657577B7320B0A3376EA51FD420D18A8"
+	],
+	"psigs": [
+		"E5C1CBD6E7E89FE9EE30D5F3B6D06B9C218846E4A1DEF4EE851410D51ABBD850",
+		"9BC470F7F1C9BC848BDF179B0023282FFEF40908E0EF88459784A4355FC86D0C",
+		"D5D8A09929BA264B2F5DF15ACA1CF2DEFA47C048DF0C3232E965FFE2F2831B1D",
+		"A915197503C1051EA77DC91F01C3A0E60BFD64473BD536CB613F9645BD61C843",
+		"99A144D7076A128022134E036B8BDF33811F7EAED9A1E48549B46D8A63D64DC9",
+		"716A72A0C1E531EBB4555C8E29FD35C796F4F231C3B039193D7E8D7AEFBDF5F7",
+		"06B6DD04BC0F1EF740916730AD7DAC794255B161221719765BDE9686A26633DC",
+		"BF6D85D4930062726EBC6EBB184AFD68DBB3FED159C501989690A62600D6FBAB",
+		"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141"
+	],
+	"msg": "599C67EA410D005B9DA90817CF03ED3B1C868E4DA4EDF00A5880B0082C237869",
+	"aggnonces": [
+		"023B11E63E2460E5E0F1561BB700FEA95B991DD9CA2CBBE92A3960641FA7469F6702CA4CD38375FE8BEB857C770807225BFC7D712F42BA896B83FC71138E56409B21",
+		"03F98BEAA32B8A38FE3797C4E813DC9CE05ADBE32200035FB37EB0A030B735E9B6030E6118EC98EA2BA7A358C2E38E7E13E63681EEB683E067061BF7D52DCF08E615",
+		"026491FBCFD47148043A0F7310E62EF898C10F2D0376EE6B232EAAD36F3C2E29E303020CB17D168908E2904DE2EB571CD232CA805A6981D0F86CDBBD2F12BD91F6D0",
+		"000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+	],
```

Comment:
Removed from json files and tests (now it just aggregates the pubnonces and just uses that, instead of checking that the aggregated result matches the one in the json)

Reply:
Sorry, I didn't mean to remove the `assert`s. I think they're useful. But they seem to be entirely independent of this aggnonces array. I suggest to remove just this `aggnonces` array because it is unused in the code.

Reply:
I think the `aggnonces` array was being used, see 7a3daa2fe7b8a22455d92ea8de197ba7cfd98ee1 which restores it to how it was before.

Reply:
The aggnonces are not being used as demonstrated by this branch: https://github.com/jonasnick/bips/commits/json-vectors-jn.

Reply:
LOL ok thanks, I definitely missed that